CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## [Unreleased](https://github.com/OpenLineage/OpenLineage/compare/0.5.1...HEAD)
+### Added
+* Airflow integration uses new TaskInstance listener API for Airflow 2.3+ [@mobuchowski](https://github.com/mobuchowski)
```

Comment:
Can we link out to any helpful Airflow docs in the changelog on the `TaskInstance` listener?

Reply:
Fixed.